docs/tools/cyberway.eosio-abidiff.md

```diff
@@ -0,0 +1,30 @@
+ \defgroup md_tools Tools
+
+ \defgroup cyberway-eosio-abidiff CyberWay-Eosio Abidiff
+ \ingroup md_tools
+
+
+# cyberway.eosio-abidiff tool
```

Comment:
There is `cyberway-abidiff` and `eosio-abidiff`. It's need to write about difference between it.